frontend/packages/core/src/AppLayout/user.tsx

```diff
@@ -76,34 +109,52 @@ const UserInformation: React.FC = () => {
   }
 
   return (
-    <Grid container alignItems="center" justify="flex-end">
-      <div>{userId()}</div>
+    <Box>
       <UserPhoto
         ref={anchorRef}
         edge="end"
         aria-controls={open ? "account-options" : undefined}
         aria-haspopup="true"
-        onClick={handleToggle}
+        onMouseEnter={handleToggle}
       >
-        <Avatar>
-          <Initials variant="h6">{userId().slice(0, 2).toUpperCase()}</Initials>
-        </Avatar>
+        <UserAvatar />
       </UserPhoto>
-      <Popper open={open} anchorEl={anchorRef.current} role={undefined} transition>
+      <Popper
+        open={open}
+        anchorEl={anchorRef.current}
+        role={undefined}
+        transition
+        popperOptions={{
+          modifiers: {
+            offset: {
+              offset: "-115,5",
```

Comment:
The popper should be right aligned with the user photo 
![Screen Shot 2020-12-01 at 11 43 58 AM](https://user-images.githubusercontent.com/1004789/100788851-84d69d80-33ca-11eb-8047-ecb05e1a8796.png)

Reply:
Is this better? i moved it more to the right. i haven't had luck getting the alignment correct with normal css properties, and I found this workaround https://github.com/lyft/clutch/pull/696#discussion_r532778102.
<img width="978" alt="Screen Shot 2020-12-02 at 12 47 16 AM" src="https://user-images.githubusercontent.com/39421794/100835092-08c86e00-343b-11eb-928f-308511d3b60f.png">

Reply:
also I left a note on figma that the white-ish avatar background doesn't translate well on the menu